pulsar-broker-auth-sasl/src/main/java/org/apache/pulsar/broker/authentication/AuthenticationProviderSasl.java

```diff
@@ -110,6 +117,10 @@ public void initialize(ServiceConfiguration config) throws IOException {
             throw new IllegalArgumentException(msg);
         }
         this.signer = new SaslRoleTokenSigner(secret);
+        this.authStates = Caffeine.newBuilder()
+                .expireAfterWrite(
+                        ServiceConfigurationUtils.getLongPropertyOrDefault(config, AUTHENTICATION_SASL_PREFIX,
```

Comment:
Why the prefix name is used to get the SASL expiration value? Maybe we need another name like `saslStatesExpirationInMillis`?